infra/main.bicep

```diff
@@ -531,6 +531,7 @@ module web './app/web.bicep' = if (hostingModel == 'code') {
       AZURE_SPEECH_SERVICE_REGION: location
       AZURE_SPEECH_RECOGNIZER_LANGUAGES: recognizedLanguages
       ORCHESTRATION_STRATEGY: orchestrationStrategy
+      CHAT_CONVERSATION_TYPE: chatConversationType
```

Comment:
Don't you need to define a param in the main.bicep too - when you do, can you add Options of [byod, custom] (or whatever makes sense

Reply:
yes, that's wip.